webview-ui/src/CreateCluster/CreateCluster.module.css

```diff
@@ -2,13 +2,85 @@
     display: grid;
     grid-template-columns: 8rem 20rem 8rem;
     grid-gap: 1rem;
+    margin-top: 18px;
+}
+.presetHeader {
+    width: 208px;
+    height: 18px;
+    margin-bottom: 10px;
+}
+
+.presetContainer {
+    width: 217px;
+    height: 166px;
+    flex-shrink: 0;
+    border-radius: 8px;
+    border: 1px solid var(--base-base-10, #808080);
```

Comment:
We should avoid hard-coding colours. There are [lots of variables](https://github.com/Azure/vscode-aks-tools/blob/main/webview-ui/src/manualTest/vars.css) we can choose from for this.

Reply:
PR https://github.com/hsubramanianaks/vscode-aks-tools/pull/4 take care of this.